File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.05.java
import java.util.Scanner;

public class Exercicio5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Repita o exercício anterior, sabendo que os números são diferentes,
        // imprimindo qual é o maior e o menor dos números.

        System.out.print("Digite o valor do primeiro número: ");
        int num1 = scanner.nextInt();

        System.out.print("Digite o valor do segundo número: ");
        int num2 = scanner.nextInt();

        if (num1 == num2) {
            System.out.println("Os números são iguais");
        } else {
            System.out.println("Os números são diferentes");

            if (num1 < num2) {
                System.out.println(num1 + " é menor que " + num2);
            } else {
                System.out.println(num1 + " é maior que " + num2);
            }
        }
    }
}

File: Livro - Módulo Python/3 - Estrutura de Decisão/Exercicio.05.py
#5.	Repita o exercício anterior sabendo que os números são diferentes, qual é o maior e o menor dos números.

numero1 = input('Digite o numero 1: ')
numero2 = input('Digite o numero 2: ')
print(' ')
if numero1==numero2:
 print('Os números são iguais')
else:
 print('Os números são diferentes')   
print(' ')

if numero1<numero2:
 print(numero1,('é menor que'),numero2)

if numero1>numero2:
 print(numero1,('é maior que'),numero2)